sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonScanner.java

```diff
@@ -87,8 +87,12 @@ public PythonScanner(
     this.parser = PythonParser.create();
 
     // computes "globalSymbolsByModuleName"
+    long startTime = System.currentTimeMillis();
+    LOG.debug("Starting to build project level symbol table");
```

Comment:
Given that we already have a message saying `Starting global symbols computation` , do we really need this?

Reply:
Indeed we don't!